planning/behavior_velocity_planner/src/utilization/util.cpp

```diff
@@ -244,6 +244,78 @@ double calcJudgeLineDistWithJerkLimit(
   return std::max(0.0, x1 + x2 + x3);
 }
 
+double findReachTime(
+  const double j, const double a, const double v, const double d, const double min,
+  const double max)
+{
+  auto f = [](const double t, const double j, const double a, const double v, const double d) {
+    return j * t * t * t / 6.0 + a * t * t / 2.0 + v * t - d;
+  };
+  const double eps = 1e-5;
+  const int max_iter = 100;
+  double lower = min;
+  double upper = max;
+  double t;
+  int iter = 0;
+  for (int i = 0; i < max_iter; i++) {
+    t = 0.5 * (lower + upper);
+    const double fx = f(t, j, a, v, d);
+    // std::cout<<"fx: "<<fx<<" up: "<<upper<<" lo: "<<lower<<" t: "<<t<<std::endl;
+    if (std::abs(fx) < eps) {
+      break;
+    } else if (fx > 0.0) {
+      upper = t;
+    } else {
+      lower = t;
+    }
+    iter++;
+  }
+  // std::cout<<"iter: "<<iter<<std::endl;
+  return t;
+}
+
+double calculateMaxSlowDownVelocity(
```

Comment:
`calculateMaxSlowDownVelocityAtTargetPoint`
This name may be easier to understand the function?

Reply:
[5950d2e](https://github.com/autowarefoundation/autoware.universe/pull/388/commits/5950d2e8b0a757c72f6a3ebc553d12aaf47ce503)